[2022-09-26T12:22:06.580+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2022-02-16T00:00:00+00:00 [queued]>
[2022-09-26T12:22:06.596+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2022-02-16T00:00:00+00:00 [queued]>
[2022-09-26T12:22:06.597+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-09-26T12:22:06.597+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 2
[2022-09-26T12:22:06.597+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-09-26T12:22:06.629+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): model.migrate_to_dbt.cast_obj> on 2022-02-16 00:00:00+00:00
[2022-09-26T12:22:06.638+0000] {standard_task_runner.py:54} INFO - Started process 1086 to run task
[2022-09-26T12:22:06.643+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_dag', 'model.migrate_to_dbt.cast_obj', 'scheduled__2022-02-16T00:00:00+00:00', '--job-id', '1721', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmpmsvok04w']
[2022-09-26T12:22:06.644+0000] {standard_task_runner.py:83} INFO - Job 1721: Subtask model.migrate_to_dbt.cast_obj
[2022-09-26T12:22:06.646+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/dbt_dag.py
[2022-09-26T12:22:06.832+0000] {task_command.py:384} INFO - Running <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2022-02-16T00:00:00+00:00 [running]> on host 3c654011b3f0
[2022-09-26T12:22:06.980+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=isaaclucky88@gmail.com
AIRFLOW_CTX_DAG_OWNER=astronomer
AIRFLOW_CTX_DAG_ID=dbt_dag
AIRFLOW_CTX_TASK_ID=model.migrate_to_dbt.cast_obj
AIRFLOW_CTX_EXECUTION_DATE=2022-02-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-16T00:00:00+00:00
[2022-09-26T12:22:06.982+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-09-26T12:22:06.983+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            cd /opt/***/migrate_to_dbt &&\n            dbt --no-write-json run --target prod --models cast_obj\n            ']
[2022-09-26T12:22:07.013+0000] {subprocess.py:86} INFO - Output:
[2022-09-26T12:22:07.017+0000] {subprocess.py:93} INFO - /bin/bash: line 3: dbt: command not found
[2022-09-26T12:22:07.018+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2022-09-26T12:22:07.043+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-09-26T12:22:07.051+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=dbt_dag, task_id=model.migrate_to_dbt.cast_obj, execution_date=20220216T000000, start_date=20220926T122206, end_date=20220926T122207
[2022-09-26T12:22:07.077+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 1721 for task model.migrate_to_dbt.cast_obj (Bash command failed. The command returned a non-zero exit code 127.; 1086)
[2022-09-26T12:22:07.095+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-09-26T12:22:07.287+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
